ignite/engine/engine.py

```diff
@@ -118,18 +117,18 @@ def compute_mean_std(engine, batch):
 
     """
 
-    _state_dict_all_req_keys = ("epoch_length", "max_epochs")
-    _state_dict_one_of_opt_keys = ("iteration", "epoch")
+    _state_dict_all_req_keys = ("epoch_length",)
```

Comment:
But if we interrupted the engine during the first epoch, we would not have `epoch_length`.